package.json

```diff
@@ -21,7 +21,8 @@
   "scripts": {
     "start": "node pokemon-showdown start",
     "build": "node build",
-    "test": "eslint --cache . && tslint --project . && node build && mocha && tsc",
+    "test": "eslint --cache . && tslint --project . && node build && mocha -i -g \"\\(slow\\)\" && tsc",
+    "fulltest": "eslint --cache . && tslint --project . && node build && mocha && tsc",
     "tsc": "tsc",
     "lint": "eslint --cache ."
```

Comment:
What about changing `"lint"` to `"eslint ---cache .&& tslint --project ."`? I would then do:

```json
"pretest": "npm run lint",
"test": "mocha -i -g \"\\(slow\\)\"",
"posttest": "npm run tsc",
"fulltest": "npm run pretest && mocha && npm run posttest",
```

`pretest` and `posttest` are special cased by `npm` to automatically run before `npm test`, but we'd need to specify them manually for `posttest`. The main motivation behind this change would be to remove duplication between the `"test"` and `"posttest"` scripts.